src/sysext/sysext.c

```diff
@@ -955,6 +1037,7 @@ static int verb_help(int argc, char **argv, void *userdata) {
                "     --json=pretty|short|off\n"
                "                          Generate JSON output\n"
                "     --force              Ignore version incompatibilities\n"
+               "     --no-reload          Do not reload the daemon\n"
```

Comment:
'the daemon' is a bit vague, say PID1 or systemd or so

Reply:
say "service manager"